Tech Radar 2019.md

```diff
@@ -0,0 +1,39 @@
+At all times:
+
+- browser and hardware interoperation
+- accessibility
+- responsive and progressive content
+- longevity of content
+- user-centric privacy and security
+
+2019:
+
+- WebXR Device API
+- content creation tools like Blender integration of glTF
+- frameworks like Sumerian, PotassiumES
+
+2020:
+
+- payment integration
+- hand gesture input
+- eye tracking
+
+2021:
+
+- shared anchors
+- long lived, simultanous XR apps
+- deeper OS integration
+
+2022:
+
+- shared layers / spaces
+- delarative content
+- personal agents
+- page-defined computer vision
```

Comment:
2022 for CV?  That seems somewhat far out.  We should have CV in pages next year, I think.  It's pretty straightforward.